zerver/lib/onboarding.py

```diff
@@ -131,6 +134,35 @@ def send_initial_direct_message(user: UserProfile) -> None:
         disable_external_notifications=True,
     )
 
+    custom_welcome_bot_message_string = ""
+    custom_welcome_bot_message_text = ""
+    # Send additional welcome bot message configured by administrators.
+    if custom_welcome_bot_message is not None:
+        custom_welcome_bot_message_text = custom_welcome_bot_message.strip()
+    elif (
+        user.realm.custom_welcome_bot_message_enabled
+        and user.realm.custom_welcome_bot_message is not None
```

Comment:
We should use `assert user.realm.custom_welcome_bot_message is not None`, right ?